src/main/java/console_app/event_adapters/EventController.java

```diff
@@ -16,12 +19,18 @@
 public class EventController {
 
     private final CalendarEventCreationBoundary eventAdder;
-    private final EventGetter eventGetter;
+    private final CalendarAnalyzer eventScheduler;
```

Comment:
It seems that eventScheduler is not used. Is it added for future function?

Reply:
Seems like I accidentally added it when merging main. Thanks for the catch